includes/templates/admin-auto-issuance-form.php

```diff
@@ -10,11 +10,7 @@
 require_once plugin_dir_path( __DIR__ ) . '/models/class-accredible-learndash-model-auto-issuance.php';
 require_once plugin_dir_path( __DIR__ ) . '/helpers/class-accredible-learndash-admin-form-helper.php';
 
-$accredible_learndash_courses = Accredible_Learndash_Model_Auto_Issuance::get_course_options();
-
-$accredible_learndash_issuance_current_page = isset( $_GET['page_num'] ) ? sanitize_key( wp_unslash( $_GET['page_num'] ) ) : 1; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
It should be deleted since I'm passing the references in `Accredible_Leardash_Ajax:load_issuance_form_html`. The issue is due to the assignment in `load_issuance_form_html`
```
 $accredible_learndash_issuance_current_page = self::get_request_value( 'page', null ); // <= `page_num`